content/en/docs/tasks/extend-kubectl/kubectl-plugins.md

```diff
@@ -3,7 +3,7 @@ title: Extend kubectl with plugins
 reviewers:
 - juanvallejo
 - soltysh
-description: With kubectl plugins, you can extend the functionality of the kubectl command by adding new subcommands.
+description: Extend the functionality of the kubectl command by adding new subcommands.
```

Comment:
nit: The plugin part is the main concept. Something like:
Install, write, and distribute kubectl extension plugins.
or
Extend kubectl by creating and installing kubectl plugins.

Reply:
Think this is good to fix after we merge?

Reply:
I've taken the second suggestion.